src/main/java/com/theironyard/controllers/UserController.java

```diff
@@ -0,0 +1,71 @@
+package com.theironyard.controllers;
+
+import com.theironyard.commands.LoginCommand;
+import com.theironyard.commands.RegisterCommand;
+import com.theironyard.entities.User;
+import com.theironyard.repositories.UserRepository;
+import com.theironyard.utilities.PasswordStorage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import javax.servlet.http.HttpSession;
+
+@Controller
+public class UserController {
+    public static final String SESSION_USERNAME = "currentUsername";
+    /* *
+     *
+     * */
+    @Autowired
+    UserRepository userRepo;
+
+    @RequestMapping(path = "/login", method = RequestMethod.GET)
+    public String getLoginPage(){
+        return "login";
+    }
+
+    @RequestMapping(path = "/register", method = RequestMethod.GET)
+    public String getRegisterPage(){
+        return "register";
+    }
+
+    /* *
```

Comment:
Nice however doc strings are `/**`  If you do it that way Intellij will automatically populate the parameters and the return statement for you